src/Tasks/FindInList.cs

```diff
@@ -18,17 +18,12 @@ public class FindInList : TaskExtension
         // The list to search through
         private ITaskItem[] _list;
         // Whether to match just the file part, or the full item spec
-        private bool _matchFileNameOnly = false;
```

Comment:
Bunch of dangling comments wherever this refactoring ran. Sad the duplicated comments were there in the first place.

Reply:
Fixed in following PR coming today/tomorrow, if that's OK